modules/update-upgrading-oc-adm-upgrade-status.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-status_{context}"]
+= Retrieving information about a cluster update using oc adm upgrade status (Technology Preview)
+
+When updating your cluster, it is useful to understand how your update is progressing. While the `oc adm upgrade` command returns limited information about the status of your update, this release introduces the `oc adm upgrade status` command as a Technology Preview feature. This command decouples status information from the `oc adm upgrade` command and provides specific information regarding a cluster update, including the status of the control plane and worker node updates.
+
+The `oc adm upgrade status` command is read-only and will never alter any state in your cluster. 
+
+[IMPORTANT]
+====	
+The `oc adm upgrade status` command is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features - Scope of Support].
+====
+
+The `oc adm upgrade status` command can be used for clusters from version 4.12 up to the latest supported release. 
+
+[IMPORTANT]
+====
+While your cluster does not need to be a TechPreview-enabled cluster, you must enable the `oc adm upgrade status` Technology Preview environment variable, otherwise the `oc` will not recognize the command and you will not be able to use the feature. 
+====
+
+.Procedure 
+
+* To enable the `oc adm upgrade status` Technology Preview environment variable, set the `OC_ENABLE_CMD_UPGRADE_STATUS` environmental variable to `true` by running the following command:
```

Comment:
Does the environment variable need to be passed with the command every time it is run (or maybe that's just how we'd prefer to show it being run for now?), or can you set the environment variable globally and then just run `oc adm upgrade status` by itself?

If the former, I wonder if the following would be more accurate as far as what running this command is fully doing; that is, that it's both setting the variable (for just this run?) and actually running the command at the same time (and this would also match the procedure module's heading more closely):
```suggestion
* To retrieve information about a cluster update with the `oc adm upgrade status` command, enable the `oc adm upgrade status` Technology Preview environment variable by setting `OC_ENABLE_CMD_UPGRADE_STATUS` to `true` while running the command:
```
If the latter, an alternative version of this could maybe be a step 1 shows setting `OC_ENABLE_CMD_UPGRADE_STATUS=true` globally, and step 2 just runs `oc adm upgrade status` by itself, followed by some example output.

Reply:
> Does the environment variable need to be passed with the command every time it is run (or maybe that's just how we'd prefer to show it being run for now?), or can you set the environment variable globally and then just run oc adm upgrade status by itself?

You can definitely set it globally and then run the command alone. Whether you do so or pass it for a single execution is simply a user preference. I agree that maybe presenting it as two separate steps may be better.

Reply:
A benefit to doc'ing as one line is that it's easier to narrowly copy/paste, without having to scroll up and see that you also need to export the environment variable in a separate command.  But :shrug:, I dunno if convenient copy-pasting is worth the trouble of trying to explain the one-line form, vs. spreading out into two separate commands.